app/routes/_index.tsx

```diff
@@ -1,50 +1,671 @@
+          <input
+            type="email"
+            id="email"
+            className="shadow-sm bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500 dark:shadow-sm-light"
+            placeholder="name@flowbite.com"
+            required
+          />
+        </div>
+        <div className="mb-5">
+          <label
+            htmlFor="text"
+            className="block mb-2 text-sm font-medium text-gray-900 dark:text-white"
           >
-            30m Tutorial
-          </a>
-        </li>
-        <Button>Hello</Button>
-        <li>
-          <a
-            className="text-blue-700 underline visited:text-purple-900"
-            target="_blank"
-            href="https://remix.run/docs"
-            rel="noreferrer"
+            Name
+          </label>
+          <input
+            type="text"
+            id="text"
+            className="shadow-sm bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500 dark:shadow-sm-light"
+            required
+          />
+        </div>
+        <div className="mb-5">
+          <label
+            htmlFor="message"
+            className="block mb-2 text-sm font-medium text-gray-900 dark:text-white"
+          >
+            Your message
+          </label>
+          <textarea
+            id="message"
+          
+            className="block p-2.5 w-full text-sm text-gray-900 bg-gray-50 rounded-lg border border-gray-300 focus:ring-blue-500 focus:border-blue-500 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500 mb-10"
+            placeholder="Leave Link comment..."
+          ></textarea>
+
+          <button
+            type="submit"
+            className="text-white w-full bg-orange-600 hover:bg-orange-800 focus:ring-4 focus:outline-none focus:ring-orange-300 font-medium rounded-lg text-sm px-5 py-2.5 text-center "
           >
-            Remix Docs
-          </a>
-        </li>
-      </ul>
-    </div>
+            Send
+          </button>
+        </div>
+      </form>
+
+      <footer className="bg-gray-100 dark:bg-gray-900 mt-10">
+        <div className="mx-auto w-full max-w-screen-xl p-4 py-6 lg:py-8">
+          <div className="md:flex md:justify-evenly">
+            <div className="mb-6 md:mb-0 mr-10">
+              <Link to="/" className="flex items-center justify-center">
+                <div className="flex flex-col">
+                  <span className="self-center text-2xl font-semibold whitespace-nowrap dark:text-white mb-5">
+                    BoilerPlate
```

Comment:
Why are you creating the footer yourself, please look for the footer and nav bar components and use those instead